pallets/inflation/src/tests.rs

```diff
@@ -105,6 +106,74 @@ fn force_inflation_fails_due_to_unprivileged_origin() {
     })
 }
 
+#[test]
+fn force_readjust_config_works() {
+    ExternalityBuilder::build().execute_with(|| {
+        // Init values
+        let old_params = InflationParams::<Test>::get();
+        let old_config = ActiveInflationConfig::<Test>::get();
+
+        // Change params, modifying staker reward parts
+        let mut new_params = old_params;
+        new_params.base_stakers_part = Perquintill::from_percent(10);
+        new_params.adjustable_stakers_part = Perquintill::from_percent(50);
```

Comment:
Question: Why not 55%? (next value)

Reply:
Ok noted for this test, I thought you wanted to use upcoming values after upgrade (10% & 55%), also because Astar live uses 65%.

Reply:
No, I'm not testing against those exact values.